docs/ENVIRONMENT.md

```diff
@@ -97,6 +97,9 @@ All tools:
   for example in systemd-nspawn, will be logged to the audit log, if the current
   kernel version supports this.
 
+* `$SYSTEMD_PARSE_EFI=0` — if set, several settings, e.g., log related settings,
+  in EFI option are ignored.
+
```

Comment:
Maybe merge this with the patch that introduces the variable? It's small enough...